datadog/resource_datadog_downtime_test.go

```diff
@@ -130,7 +134,7 @@ func TestAccDatadogDowntime_BasicMultiScope(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"datadog_downtime.foo", "message", "Example Datadog downtime message."),
 					resource.TestCheckResourceAttr(
-						"datadog_downtime.foo", "monitor_tags.0", "*"),
+						"datadog_downtime.foo", "monitor_tags.2679715827", "*"),
```

Comment:
I am not sure i understand these changes here. Why do we have to change assertions?

Reply:
Since it's not a list anymore, the keys aren't zero indexed and these values are hashes of the values. See https://github.com/terraform-providers/terraform-provider-datadog/commit/6552bcee02a611b4b36e8f7dcdbc1622838bb921#diff-01173e30cd220a18166d845ecb9bcd11R106